inc/core/styles/frontend.php

```diff
@@ -978,7 +978,7 @@ public function setup_blog_meta() {
 		list( $context, $allowed_context ) = $this->get_cpt_context();
 		$archive_avatar_size_meta_key      = Config::MODS_ARCHIVE_POST_META_AUTHOR_AVATAR_SIZE;
 		$single_avatar_size_meta_key       = Config::MODS_SINGLE_POST_META_AUTHOR_AVATAR_SIZE;
-		if ( in_array( $context, $allowed_context, true ) && neve_is_new_skin() && is_singular( $context ) || is_post_type_archive( $context ) ) {
+		if ( in_array( $context, $allowed_context, true ) && is_singular( $context ) || is_post_type_archive( $context ) ) {
 			$archive_avatar_size_meta_key = 'neve_' . $context . '_archive_author_avatar_size';
 			$single_avatar_size_meta_key  = 'neve_single_' . $context . '_avatar_size';
```

Comment:
we should have keep in mind, https://github.com/Codeinwp/neve/pull/3830 PR should be released or before with this PR

Reply:
Here it does not matter, the check is redundant as we already check for new_skin on L973 and it will bail if it is not new_skin, hence the check here is not required.

Reply:
you're right, I've missed that, sorry.